2. Services/2.2 Processings/2.2 Processings.md

```diff
@@ -27,34 +27,35 @@ TryCatch(async () =>
+When used, Processing services live between foundation services and the rest of the application. They may not call Entity or Business brokers, but they may call Utility brokers such as logging brokers, time brokers, and any other brokers that offer supporting functionality and are not specific to any particular business logic. Here's a visual of where Processing services are located on the map of our architecture:
 
 <br />
     <p align="center" >
         <img src="https://user-images.githubusercontent.com/1453985/111928442-d9cd1b00-8a70-11eb-82c8-83399c98c87d.png" />
     </p>
 <br />
 
-On the right side of a Processing service lies all the non-local models and functionality, whether it's through the brokers, or the models that the foundation service is trying to map into local models.
-On the left side of Processing services is pure local functionality, models and architecture. Starting from the Processing services themselves, there should be no trace or track of any native or non-local models in the system.
+On the right side of a Processing service lies all the non-local models and functionality, whether through the brokers or the models the foundation service is trying to map into local models.
 
-## 2.2.2 Charachteristics
+On the left side of Processing, services are pure local functionality, models, and architecture, and there should be no trace or track of any native or non-local models in the system.
 
-Processing services in general are combiners of multiple primitive-level functions to produce a higher-order business logic. but they have much more characteristics than just that, let's talk about those here.
+## 2.2.2 Characteristics
+
+Processing services, in general, are combiners of multiple primitive-level functions to produce a higher-order business logic. But they have more characteristics than just that. Let's talk about those here.
 
 ### 2.2.2.0 Language
 
-The language used in processing services defines the level of complexity and the capabilities it offers.
-Usually, processing services combine two or more primitive operations from the foundation layer to create a new value.
+The language used in Processing services defines the complexity level and its capabilities. Usually, Processing services combine two or more primitive operations from the foundation layer to create a new value.
 
 #### 2.2.2.0.0 Functions Language
 
-At a glance, Processing services language change from primitive operations such as `AddStudent` or `RemoveStudent` to `EnsureStudentExists` or `UpsertStudent`. they usually offer a more advanced business-logic operations to support a higher order functionality.
-Here's some examples for the most common combinations a processing service may offer:
+At a glance, Processing services language change from primitive operations such as `AddStudent` or `RemoveStudent` to `EnsureStudentExists` or `UpsertStudent`. they usually offer more advanced business-logic operations to support a higher order functionality.
```

Comment:
Should this be `Processing services' language`?

Reply:
@dlandi friendly reminder on this one

Reply:
We're talking about a  general case here versus a specific case. I could make it possessive so that it matches most of your other paragraphs. I'll do that. 

Aslo, I could put a "can" in the for more clarity: 

> Processing services' _**can**_ language change

Give me a second to update this PR.

Reply:
@dlandi sounds good. let me know when you're ready.